client/jetpack-cloud/sections/partner-portal/assign-license-step-progress/index.tsx

```diff
@@ -48,14 +48,14 @@ export default function AssignLicenseStepProgress( { currentStep }: Props ) {
 	const paymentMethodRequired = useSelector( doesPartnerRequireAPaymentMethod );
 	const sites = useSelector( getSites ).length;
 
-	const steps: Step[] = [ { key: 'issueLicense', label: translate( 'Issue new license' ) } ];
+	const steps: Step[] = [ { key: 'issueLicense', label: translate( 'Issue new licenses' ) } ];
 
 	if ( paymentMethodRequired ) {
 		steps.push( { key: 'addPaymentMethod', label: translate( 'Add Payment Method' ) } );
 	}
 
 	if ( sites > 0 ) {
-		steps.push( { key: 'assignLicense', label: translate( 'Assign license' ) } );
+		steps.push( { key: 'assignLicense', label: translate( 'Assign licenses' ) } );
```

Comment:
I guess this change is copied from the previous base branch, which we decided to add later since multiple license flow is not enabled in production.